backend/app/techniques/base_technique.py

```
@@ -0,0 +1,93 @@
+        last_error = None
+
+        for attempt, delay in enumerate(RETRY_DELAYS):
+            try:
+                raw_response = await self._call_llm(state, item_id)
+                parsed = self._parse_response(raw_response, item_id)
+                result.item_scores = parsed["item_scores"]
+                result.trace_events = parsed["trace_events"]
+                result.token_usage = parsed.get("token_usage", {})
+                result.cost_usd = parsed.get("cost_usd", 0.0)
+                result.success = True
+                return result
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)